dsp/modules/sentence_vectorizer.py

```diff
@@ -1,6 +1,6 @@
 import abc
 from typing import List, Optional
-
+import boto3
```

Comment:
Update with try/except block like in [aws_providers](https://github.com/stanfordnlp/dspy/blob/bb97c72da6312600b8ac1a5228f79b07afc427b1/dsp/modules/aws_providers.py#L8).